.github/workflows/commit_enclave_tls.yml

```diff
@@ -0,0 +1,118 @@
+        echo "deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main" | tee /etc/apt/sources.list.d/intel-sgx.list && wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -;
+        apt-get update -y && apt-get install -y libsgx-dcap-quote-verify libsgx-dcap-default-qpl libsgx-dcap-ql-dev libsgx-uae-service;
+        cp -rf /root/inclavare-containers/${{ matrix.tag }}/enclave-tls /opt'
+
+    - name: Install centos dependency
+      if: ${{ contains(matrix.tag, 'centos') }}
+      run: |
+        docker exec $inclavare_test bash -c 'cd /root/inclavare-containers/${{ matrix.tag }};
+        yum -y install yum-utils wget iptables;
+        wget -c https://download.01.org/intel-sgx/sgx-linux/2.13/distro/centos8.2-server/sgx_rpm_local_repo.tgz;
+        tar xzf sgx_rpm_local_repo.tgz;
+        yum-config-manager --add-repo sgx_rpm_local_repo;
+        yum makecache;
+        yum install --nogpgcheck -y libsgx-dcap-quote-verify libsgx-dcap-default-qpl libsgx-dcap-ql-devel libsgx-uae-service;
+        rm -f sgx_rpm_local_repo.tgz;
+        mv /root/inclavare-containers/${{ matrix.tag }}/enclave-tls /opt'
+
+    - name: Install alinux dependency
+      if: ${{ contains(matrix.tag, 'alinux') }}
+      run: docker exec $inclavare_test bash -c 'cd /root/inclavare-containers/${{ matrix.tag }};
+        yum -y install yum-utils wget tar procps alinux-release-experimentals  protobuf-c-devel;
+        wget -c https://mirrors.openanolis.org/inclavare-containers/alinux2/sgx_rpm_local_repo.tar.gz;
+        tar xzf sgx_rpm_local_repo.tar.gz;
+        yum-config-manager --add-repo file:/root/inclavare-containers/${{ matrix.tag }}/sgx_rpm_local_repo;
+        yum makecache;
+        yum install --nogpgcheck -y libsgx-dcap-quote-verify libsgx-dcap-default-qpl libsgx-dcap-ql-devel libsgx-uae-service;
+        rm -f sgx_rpm_local_repo.tar.gz;
+        mv /root/inclavare-containers/${{ matrix.tag }}/enclave-tls /opt'
+
+    - name: Run enclave tls server for mutual sgx_la
+      run: docker exec $inclavare_test bash -c 'cd /opt/enclave-tls/bin/ && ./enclave-tls-server -a sgx_la -m' &
+
+    - name: Run enclave tls client for mutual sgx_la
+      run: docker exec $inclavare_test bash -c 'cd /opt/enclave-tls/bin/ && ./enclave-tls-client -l debug -a sgx_la -m'
+        docker restart $inclavare_test;
+
+    - name: Run enclave tls server for mutual sgx_ecdsa
+      if: ${{ contains(matrix.sgx, 'SGX2') }}
+      run: |
+        docker cp /etc/sgx_default_qcnl.conf  $inclavare_test:/etc/;
+        docker exec $inclavare_test bash -c 'cd /opt/enclave-tls/bin/ && ./enclave-tls-server -l debug -a sgx_ecdsa -p 1236 -m' &
+
+    - name: Run enclave tls client for mutual sgx_ecdsa
+      if: ${{ contains(matrix.sgx, 'SGX2') }}
+      run: docker exec $inclavare_test bash -c 'cd /opt/enclave-tls/bin/ && ./enclave-tls-client -l debug -a sgx_ecdsa -p 1236 -m'
```

Comment:
Can you add a check for the test result? e.g, check the key words "Hello and welcome to ..."